examples/basic_gazelle/go.mod

```diff
@@ -0,0 +1,10 @@
+// Go uses this file to track module dependencies and to mark the root directory
+// of a Go module. This file is maintained with Go tools outside of Bazel.
```

Comment:
This could also point to `@rules_go//go`. 

Now that I think about it, we could add another alias so that users can `bazel run @rules_go`.

Reply:
Done.

I kind of like the explicitness of `@rules_go//go`. If it were just `@rules_go`, I think it would be less clear that it's running the Go command. But maybe I haven't used it enough yet to be annoyed by the extra typing.